web-ts/src/routes/__tests__/PoisPage.spec.tsx

```diff
@@ -84,11 +89,13 @@ describe('PoisPage', () => {
   it('should render a not found error', () => {
     const city = cities[0]
     const language = languages[0]
-    mockUseLoadFromEndpointOnce(() => ({
+    const useLoadFromEndpointMock = (() => ({
       data: pois,
       loading: false,
-      error: null
-    }))
+      error: null,
+      refresh: () => null
+    })) as typeof useLoadFromEndpoint
+    mockUseLoadFromEndpointOnce(useLoadFromEndpointMock)
```

Comment:
Maybe we could move this to a helper method/create a global mock for this.

Reply:
done